core/src/main/scripts/importer/validateData.py

```diff
@@ -1700,6 +1700,14 @@ def checkLine(self, data):
                         extra={'line_number': self.line_number,
                                'column_number': col_index + 1,
                                'cause': value})
+                # this one gives problems in old parts of the java code such as CnaJSON.processCnaFractionsRequest(),
+                # so block commas in sample id:
+                if ',' in value:
+                    self.logger.error(
+                        'Comma (,) in SAMPLE_ID is not supported',
+                        extra={'line_number': self.line_number,
+                               'column_number': col_index + 1,
+                               'cause': value})
```

Comment:
I tested this and it works well. I've tested some other symbols, `+ / = *` and the `/ = *` hide the whole sample in the Clinical Data tab. The sample with `+` was displayed correctly without the `+`.

I didn't experience issues with the CNA table.